library/src/tensor_operation_instance/gpu/CMakeLists.txt

```
@@ -183,6 +183,10 @@ FOREACH(subdir_path ${dir_list})
             message("bf8 instance found!")
             set(add_inst 1)
         endif()
+        if(("${cmake_instance}" MATCHES "_bf16" OR "${cmake_instance}" MATCHES "_b16") AND DTYPES MATCHES "bf16")
+        message("bf16 instance found!")
+        set(add_inst 1)
```

Should this be indented (like the other `if` blocks)?
```suggestion
            message("bf16 instance found!")
            set(add_inst 1)
```